apps/src/applab/designElements/MLModelSelectorRow.jsx

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import {connect} from 'react-redux';
+import {actions} from '../redux/applab';
+
+class MLModelSelectorRow extends React.Component {
+  static propTypes = {
+    modelIds: PropTypes.array,
+    selectedId: PropTypes.string,
+    setMLModelId: PropTypes.function
+  };
+
+  handleChange = event => {
+    event.preventDefault();
+    this.props.setMLModelId(event.target.value);
+  };
+
+  render() {
+    return (
+      <div>
+        <select
+          onChange={event => this.handleChange(event)}
+          value={this.props.selectedId}
+        >
+          {this.props.modelIds.map((modelId, index) => {
+            return (
+              <option key={index} value={modelId}>
+                {modelId}
+              </option>
+            );
+          })}
+        </select>
+      </div>
+    );
+  }
+}
+
+export default connect(
+  state => ({
+    selectedId: state.mlModelDetails.modelId
+  }),
+  dispatch => ({
+    setMLModelId(modelId) {
```

Comment:
I think if you go down the redux route you'll need another param to distinguish which predict panel you're updating in case a student adds more than one to their app.

Reply:
+1 to jillian's comment. however, is there a reason we want/need to use redux to store this state? it looks like the existing design elements store their state in the component rather than redux.

if only `<PropertyRow/>` and `<MLModelSelectorRow/>` need to access this state, i'd have PropertyRow manage the state and pass it as props into MLModelSelectorRow